mappings/net/minecraft/world/biome/layer/IdentitySamplingLayer.mapping

```diff
@@ -1,4 +1,4 @@
 CLASS cms net/minecraft/world/biome/layer/IdentitySamplingLayer
 	METHOD a sample (Lcln;I)I
 		ARG 1 context
-		ARG 2 value
+		ARG 2 previousBiome
```

Comment:
This is wrong. Not all values are biomes! Some are climate IDs, other are temperatures, others are just random noise which are converted to biomes/climates based by a future layer.

Reply:
It checks if value is an ocean ID

Reply:
What checks if the value is an ocean ID? This is an interface implemented by many layers, it shouldn't be named based on just what one implementing layer does.

Reply:
I would think that means it's a previous biome?

Reply:
What? No. There are layers that take random noise as an input, so the `value` is not always a biome.

Reply:
do I need to run gradlew yarn every time I need to open enigma

Reply:
Yes

Reply:
ah. I didn't see value affects the whole interface